tsconfig.json

```diff
@@ -25,7 +25,7 @@
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
     /* Modules */
-    "module": "ES6" /* Specify what module code is generated. */,
+    "module": "es2020" /* Specify what module code is generated. */,
```

Comment:
Why this version?

Reply:
When using the ES6 version, an error appears:
The 'import.meta' meta-property is only allowed when the '--module' option is 'es2020', 'es2022', 'esnext', 'system', 'node16', or 'nodenext'.ts(1343)
![image](https://github.com/PL-FE-SEP23-REACTivation/api-products/assets/56074232/8906cc3e-abfb-493c-ac12-525762eac3ee)

Reply:
Alright, let's use it then